spk/gitea/Makefile

```diff
@@ -1,12 +1,12 @@
 SPK_NAME = gitea
-SPK_VERS = 1.15.9
+SPK_VERS = 1.15.10
 SPK_REV = 1
```

Comment:
each package update must increment the SPK_REV
```suggestion
SPK_REV = 2
```

Reply:
Done